composer/trainer/ddp.py

```diff
@@ -93,6 +126,10 @@ def __init__(self,
         self.fork_rank_0 = fork_rank_0
         self.processes: List[subprocess.Popen[str]] = []
         self.find_unused_parameters = find_unused_parameters
+        if ddp_sync_strategy is None:
+            self.ddp_sync_strategy = DDPSyncStrategy.SINGLE_AUTO_SYNC if not find_unused_parameters else DDPSyncStrategy.MULTI_AUTO_SYNC
```

Comment:
Here, shouldn't we be using `FORCED_SYNC` when find_unused_parameters is enabled

Reply:
Yeah I think I agree that makes more sense now as well.